gradoop-examples/src/main/java/org/gradoop/benchmark/grouping/GroupingBenchmark.java

```diff
@@ -293,6 +303,10 @@ private static void readCMDArguments(final CommandLine cmd) {
     OUTPUT_PATH = cmd.getOptionValue(OPTION_OUTPUT_PATH);
     CSV_PATH = cmd.getOptionValue(OPTION_CSV_PATH);
 
+    // input format
+    boolean hasInputFormat = cmd.hasOption(OPTION_INPUT_FORMAT);
```

Comment:
`hasInputFormat` define a var is not necessary because it is used only once in the line below.